roles/routeros_common/molecule/default/converge.yml

```diff
@@ -3,6 +3,15 @@
   hosts: all
   gather_facts: false
   tasks:
-    - name: Replace this task with one that validates your content
-      ansible.builtin.debug:
-        msg: "This is the effective test"
+    - name: "Include group vars for os_routeros"
+      ansible.builtin.include_vars: ../../../../group_vars/os_routeros.yml
+
+    - name: "Include hamwan.routeros_common"
+      ansible.builtin.include_role:
+        name: routeros_common
+      vars:
+        test_flaky_network: true
+        # TODO(dpk) - The following should be coming from the inventory, but aren't. Why?
```

Comment:
is it because the inventory comes from `routeros_hosts.py`? that script is not setting ansible_port and ansible_user, although ansible_ssh_private_key_file does get set there I think.

...upon closer inspection I guess I'm wrong. But I'm not really an ansible expert. the interaction/distinction between `common_vars` and `host_vars` in that script might be an issue...

Reply:
This appears to no longer be an issue and this comment and the following 3 lines can be removed.  Inventory is working.